### ISSUE TITLE
Duda servidor

### ISSUE THREAD

**Issue by FGarciaVarela** [IIC2133-PUC/2022-2#105]
Hola! Quería saber si está funcionando el servidor pues no me arroja nada (no carga), para revisar la parte 1? 
Vi que subieron unos test case para esta parte pero si no me equivoco, solo están los inputs.

Saludos!

**Comment by CarloGauss33** [comment]
Hola El servidor aun no esta habilidato. Y en los tests hay un archivo json donde se dan datos de la solucion. Lo que pasa es que la solucion no es unica. Por lo que no hay un output esperado como tal

**Comment by FGarciaVarela** [comment]
Super! No es única pues el orden de busqueda puede cambiar verdad? Tipo el ejemplo de la solución del enunciado;
16
1 5 2 5
2 5 3 3
1 5 1 1
7 7 7 3
3 3 7 3

Sería lo mismo que 
16
1 5 2 5
2 5 3 3
1 5 1 1
3 3 7 3
7 3 7 7

No?

Saludos!

**Comment by CarloGauss33** [comment]
Claro. Pero además es porque suponte podria existir mas de una configuración con el mismo costo y mismo numero maximo de aristas salientes

**Comment by FGarciaVarela** [comment]
Perfecto. Muchas gracias.

Y perdón por preguntar tanto jeje, pero necesariamente debe ir el costo primero? (Que se me ocurre algo pero se irían imprimiendo los nodos primero y sumando los valores quedando el costo al final).

Gracias por todo!

**Comment by carolinamaggi** [comment]
Sí, debe ir el costo primero, lo que podrías hacer es ir guardando las aristas e imprimirlas al final

**Comment by FGarciaVarela** [comment]
Gracias!
